_posts/en/newsletters/2022-08-10-newsletter.md

```diff
@@ -0,0 +1,125 @@
+  Core only relays individual unconfirmed transactions that pay a
+  [feerate of at least one satoshi per vbyte][topic default minimum
+  transaction relay feerates] (1 sat/vbyte).  If a node's mempool fills
+  with transactions paying at least 1 sat/vbyte, then a higher feerate
+  will need to be paid.  Transactions paying a lower feerate can still
+  be included in blocks by miners and those blocks will be relayed.
+  Other node software implements similar policies.
+
+    Lowering the default minimum feerate has been discussed in the past
+    (see [Newsletter #3][news3 min]) but [hasn't been merged][bitcoin
+    core #13922] into Bitcoin Core.  The topic saw renewed
+    [discussion][chauhan min] in the past couple weeks:
+
+    - *Individual change effectiveness:* it was [debated][todd min] by
+      [several][vjudeu min] people how effective it was for individual
+      node operators to change their policies.
+
+    - *Past failures:* it was [mentioned][harding min] that the previous
+      attempt to lower the default feerate was hampered by the lower
+      rate also reducing the cost of several minor denial-of-service
+      (DoS) attacks.
+
+    - *Alternative relay criteria:* it was [suggested][todd min2] that
+      transactions violating certain default criteria (such as the
+      default minimum feerate) could instead fulfill some separate
+      criteria made DoS attacks costly---for example, if a modest amount
+      of hashcash-style proof of work committed to the transaction to
+      relay.
+
+    The discussion did not reach a clear conclusion as of this writing.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME:LarryRuane
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/12345#l-123"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Core Lightning 0.12.0rc1][] is a release candidate for the next major
+  version of this popular LN node implementation.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25610][] opts-in the RPCs and `-walletrbf` to RBF
+  by default. This follows the update mentioned in
```

Comment:
The first sentence throws me off. I would have expected `-walletrbf` to already cause the wallet to use RBF. Do you mean that all RPCs act as if a `-walletrbf` had gotten passed along after this change?